### PR TITLE
Update links in Dockerfile / package.json / app.json / Toools

### PR DESCRIPTION
**Pull request by ewold**
Updated links from the old repo to the new 

## Description
Updated github links to reflect to the new repo

## Motivation and Context
Old Repo is gone 

## How Has This Been Tested?
Clicking and testing urls

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by hhu94** [comment]
+1

**Comment by mlindner** [comment]
+1 Please add this

File: Tools/Hex-Beehive-Generator/Instructions_for_location_generator.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-https://github.com/AHAAAAAAA/PokemonGo-Map/wiki/How-to-use-Hex-Beehive-Generator
+https://www.jonaharagon.com/PoGoMapWiki/#!beehive.md
```
Comment (invisiblek) [inline Tools/Hex-Beehive-Generator/Instructions_for_location_generator.txt]: no

**Comment by invisiblek** [comment]
Other than that one comment, this looks good

**Comment by ewold** [comment]
If you follow to link they link to that wiki. 
So whats the currect url ? 
* https://github.com/PokemonGoMap/PokemonGo-Map/wiki/How-to-use-Hex-Beehive-Generator
* https://www.jonaharagon.com/PoGoMapWiki/#!beehive.md

**Comment by invisiblek** [comment]
*sigh*

let me see what the plan here is for the wiki for sure. apparently people keep changing their damn minds...

**Pull request merged**